Update documentation

johnschoeman opened the pull request:
Why:
Some of our documentation is out of date and we would like for new users
to the project be able to onboard without the friction of stale info.

This commit:
- Updates the example.env.bt
- Updates the readme
- Removes the unused changelog
- Updates the pull request template to follow the team's conventions

aledustet reviewed (approved):
Could the example file change be a reason why it is not building right now?

johnschoeman commented:
> Could the example file change be a reason why it is not building right now?

Potentially, though that would be very odd. Ill try separating them out.

johnschoeman committed:
Update documentation

Why:
Some of our documentation is out of date and we would like for new users
to the project be able to onboard without the friction of stale info.

This commit:

Updates the readme
Removes the unused changelog
Updates the pull request template to follow the team's conventions
